Modules/Devices/src/Devices.Domain/Aggregates/PushNotifications/PnsRegistration.cs

```diff
@@ -8,25 +8,34 @@ public class PnsRegistration
 {
     private PnsRegistration() { }
 
-    public PnsRegistration(IdentityAddress identityAddress, DeviceId deviceId, PnsHandle handle, string appId)
+    public PnsRegistration(IdentityAddress identityAddress, DeviceId deviceId, PnsHandle handle, string appId, Environment environment)
     {
         IdentityAddress = identityAddress;
         DeviceId = deviceId;
         Handle = handle;
         UpdatedAt = SystemTime.UtcNow;
         AppId = appId;
+        Environment = environment;
     }
 
     public IdentityAddress IdentityAddress { get; }
     public DeviceId DeviceId { get; }
     public PnsHandle Handle { get; private set; }
     public string AppId { get; set; }
     public DateTime UpdatedAt { get; private set; }
+    public Environment? Environment { get; private set; }
```

Comment:
Shouldn't be nullable. Instead, set the default value in the database to Production, so that existing rows are updated during migration.

Reply:
Isn't this applicable to APNS registrations only?
Hence why I made it nullable.
I get that it's technically harmless to have it for fcm, although we don't do anything with it.

Reply:
Ohh, damn, you're right. I didn't think about that.

But yeah, we should probably set it for FCM as well then. I don't like having a distinction here. 😞